Comments/ClientApp/src/document-processing/__tests__/process-preview-html.test.js

```diff
@@ -1,43 +1,37 @@
-/* eslint-env jest */
-
-import { processPreviewHtml } from "../process-preview-html";
 import React from "react";
-import { mount } from "enzyme";
-
-describe("[ClientApp]", () => {
-	describe("Render Preview HTML", () => {
-
-		function setupHtml(html) {
-			return {
-				wrapper: mount(
-					<div>{processPreviewHtml(html)}</div>,
-				),
-			};
-		}
+import { render } from "@testing-library/react";
+import { processPreviewHtml } from "../process-preview-html";
 
-		const consoleErrorReset = console.error;
+const consoleErrorReset = console.error;
 
-		beforeEach(() => {
-			console.error = consoleErrorReset;
-		});
+beforeEach(() => {
+	console.error = consoleErrorReset;
+});
 
-		it("doesn't render a comment button if there's no comment in the markup", () => {
-			const instance = setupHtml(
-				"<p><a href='#'>Hey!</a> Ain't no comment anywhere <span>here</span></p>",
-			);
-			expect(instance.wrapper.find("div.ConversionError").length).toEqual(0);
-		});
+test("doesn't render a comment button if there's no comment in the markup", () => {
+	const {container} = render(
+		<div>
+			{processPreviewHtml("<p><a href='#'>Hey!</a> Ain't no comment anywhere <span>here</span></p>")}
+		</div>,
+	);
+	expect(container.getElementsByClassName("ConversionError").length).toEqual(0);
```

Comment:
Any alternative selector to classname you can check for then it's not tied to markup?